modules/containers/ecr/modulestack.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  # DeploymentName:
+  #   Type: String
+  #   Description: The name of the deployment
+  # ModuleName:
+  #   Type: String
+  #   Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "ecr:*Repository"
+              - "ecr:*LifecyclePolicy"
+            Effect: Allow
+            Resource:
+              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/*"
```

Comment:
Can we scope this to the resolved ECR repo name?

Reply:
Thanks, updated